pkg/postgres/walker/walker.go

```diff
@@ -360,13 +360,16 @@ func handleStruct(ctx context, schema *Schema, original reflect.Type) {
 			case reflect.Uint8:
 				schema.AddFieldWithType(field, Bytes, opts)
 				continue
-			case reflect.Uint32, reflect.Uint64, reflect.Int32, reflect.Int64:
+			case reflect.Int32:
 				if typeIsEnum(elemType) {
 					schema.AddFieldWithType(field, EnumArray, opts)
 				} else {
 					schema.AddFieldWithType(field, IntArray, opts)
 				}
 				continue
+			case reflect.Uint32, reflect.Uint64, reflect.Int64:
```

Comment:
I would suggest to keep using integer to store uint32, just convert it to/from int32 before and after update/write the data.

We need to support uint64 which is also beyond max int64.

Reply:
We don't currently use uint64.  I have concerns about using integer for uint32.  I feel like that is asking for trouble from a maintenance perspective as it would be easy for a conversion of uint32 to/from int32 to be missed and introduce a bug.  IMO using bigint is safer and allows us to more quickly resolve the blocker.  It also seems to be in line with how the various ORMs map the types.  the ORMs also seem to map Uint64 to bigint due to the performance degradation of using numeric.